main_objs.py

```diff
@@ -619,14 +640,16 @@ def getNgrams(self, df=None, n=3, how='columnwise', other=None, held='Held',
                     if cell_type == str:
                         combo.insert(loc=1, column='Joiner', value=', ')
                         combo['_'] = '_'
-                        combo = [combo.shift(-i) for i in range(n)]
+                        # TODO add shift option
```

Comment:
Same as above. Fine to have "# add shift option" as a comment, but take out "TODO".